src/yoroi-wallets/utils/validators.test.ts

```diff
@@ -0,0 +1,25 @@
+import {YoroiNFTModerationStatus} from '../types'
+import {isValidModerationStatus} from './validators'
+
+describe('isValidModerationStatus', () => {
+  it('should return true on valid NFT Moderation status', () => {
+    expect(isValidModerationStatus(YoroiNFTModerationStatus.PENDING)).toBe(true)
+    expect(isValidModerationStatus(YoroiNFTModerationStatus.GREEN)).toBe(true)
+    expect(isValidModerationStatus(YoroiNFTModerationStatus.RED)).toBe(true)
+    expect(isValidModerationStatus(YoroiNFTModerationStatus.MANUAL_REVIEW)).toBe(true)
+    expect(isValidModerationStatus(YoroiNFTModerationStatus.YELLOW)).toBe(true)
+  })
+
+  it('returns false on invalid NFT Moderation status', () => {
+    expect(isValidModerationStatus('invalid' as YoroiNFTModerationStatus)).toBe(false)
+  })
+
+  it('returns false when given invalid types', () => {
+    expect(isValidModerationStatus(null)).toBe(false)
```

Comment:
no need to test invalid types
unreliable data should be type-checked when parsed

Reply:
const parseModerationStatus = (data: unknown): ModerationStatus | undefined => { ... }

Reply:
oh I see now, 
we are moving away from validators.ts, and moving toward colocation

Reply:
in this case, the `isValidModerationStatus` type guard should be moved closer to parsing.ts

Reply:
I've removed the validator and added parseModerationStatus